tests/adapters/unified_tests/vlan_management_test.py

```diff
@@ -238,6 +239,13 @@ def test_vrf_management(self):
         vlan = self.get_vlan_from_list(2999)
         assert_that(vlan.vrf_forwarding, is_(none()))
 
+    @skip_on_switches("juniper", "juniper_qfx_copper", "dell", "dell_telnet", "dell10g", "dell10g_telnet", "juniper_mx", "juniper", "brocade_telnet", "brocade")
```

Comment:
twice juniper?
also is those are legacy tests so I don't think it necessary to add anything here

Reply:
Yeah a compliance test would be better

Reply:
True that...